cmd/mesos_state.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/url"
+	"strconv"
+
+	"github.com/dcos/dcos-diagnostics/util"
+	"github.com/dcos/dcos-go/dcos"
+	"github.com/spf13/cobra"
+)
+
+// daemonCmd represents the daemon command
+var stateCmd = &cobra.Command{
```

Comment:
Please add a test for this. IMHO it's enough to have one tests that just runs the cmd using a mock HTTP server and checks whether the body is just returned. To do this I would seperate out the logic from the `RunE` function into a function of its own and run the tests against that then.

Reply:
Can you take a look now?

Reply:
I would actually love to see a test that tests this command itself so that we can be sure that this command prints the expected result. In https://github.com/makkes/gitlab-cli/blob/master/cmd/issue/inspect/inspect.go#L15 I even passed an `io.Writer` to the function so that the cmd itself is only one line of code and the rest is completely testable.

I find this important esp. since we don't have e2e/integration tests for this.